airgap/methods.go

```diff
@@ -39,10 +41,16 @@ var (
 	WithdrawGold = registerMethod(registry.LockedGoldContractID.String(), "withdraw", []argParser{bigIntParser})
 
 	// Election
-	Vote               = registerMethod(registry.ElectionContractID.String(), "vote", []argParser{addressParser, bigIntParser})
-	ActivateVotes      = registerMethod(registry.ElectionContractID.String(), "activate", []argParser{addressParser})
-	RevokePendingVotes = registerMethod(registry.ElectionContractID.String(), "revokePending", []argParser{addressParser, addressParser, bigIntParser})
-	RevokeActiveVotes  = registerMethod(registry.ElectionContractID.String(), "revokeActive", []argParser{addressParser, addressParser, bigIntParser})
+	Vote                            = registerMethod(registry.ElectionContractID.String(), "vote", []argParser{addressParser, bigIntParser})
+	ActivateVotes                   = registerMethod(registry.ElectionContractID.String(), "activate", []argParser{addressParser})
+	RevokePendingVotes              = registerMethod(registry.ElectionContractID.String(), "revokePending", []argParser{addressParser, addressParser, bigIntParser})
+	RevokeActiveVotes               = registerMethod(registry.ElectionContractID.String(), "revokeActive", []argParser{addressParser, addressParser, bigIntParser})
+	CanReceiveVotes                 = registerMethod(registry.ElectionContractID.String(), "canReceiveVotes", []argParser{addressParser, bigIntParser})
+	GetEpochNumber                  = registerMethod(registry.ElectionContractID.String(), "getEpochNumber", []argParser{})
+	GetEpochSize                    = registerMethod(registry.ElectionContractID.String(), "getEpochSize", []argParser{})
+	GetEpochNumberOfBlock           = registerMethod(registry.ElectionContractID.String(), "getEpochNumberOfBlock", []argParser{bigIntParser})
+	GetActiveVotesForGroup          = registerMethod(registry.ElectionContractID.String(), "getActiveVotesForGroup", []argParser{addressParser})
+	GetActiveVotesForGroupByAccount = registerMethod(registry.ElectionContractID.String(), "getActiveVotesForGroupByAccount", []argParser{addressParser, addressParser})
```

Comment:
Can be done in a followup PR, but we'll also need a few more `ReleaseGold` reads:

`MaxDistribution = registerMethod(releaseGold, "maxDistribution", ...)`
`TotalWithdrawn = registerMethod(releaseGold, "totalWithdrawn", ...)`